circle.yml

```
@@ -2,7 +2,6 @@ version: 2
 jobs:
   build:
     machine: true
-    working_directory: ~/go/src/github.com/docker/go-connections
```

Can you try adding (from the list of https://circleci.com/docs/2.0/configuration-reference/#available-machine-images)

```suggestion
    image: ubuntu-1604:202004-01
```

---

Oops, got distracted by work and forgot about this one. Yup, I'll give it a shot.

---

Argh. Think I messed up indentation;

```
# Unable to parse YAML
# mapping values are not allowed here
#  in 'string', line 5, column 12:
#           image: ubuntu-1604:202004-01
#                ^
# 
# -------
# Warning: This configuration was auto-generated to show you the message above.
# Don't rerun this job. Rerunning will have no effect.
false
```

---

I hate YAML… sorry, I even noticed that my editor kept trying to move that around, should have listened to it.

---

Yes, looks like the "boolean" may be an old variant, and it needs to be; https://circleci.com/docs/2.0/configuration-reference/#machine

```yaml
jobs:
  build:
    machine:
      image: ubuntu-1604:202004-01
```